src/command_modules/azure-cli-locationbasedservices/azure/cli/command_modules/locationbasedservices/custom.py

```diff
@@ -0,0 +1,134 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.log import get_logger
+from knack.prompting import prompt_y_n
+from knack.util import CLIError
+
+from azure.mgmt.locationbasedservices.models import (
+    LocationBasedServicesAccountCreateParameters,
+    Sku)
+
+ACCOUNT_LOCATION = 'global'
+
+logger = get_logger(__name__)
+
+
+# pylint: disable=line-too-long
+def get_account(client, resource_group_name, account_name):
+    """Get a Location Based Services Account.
+
+    :param resource_group_name: The name of the Azure Resource Group.
+    :type resource_group_name: str
+    :param account_name: The name of the Location Based Services Account.
+    :type account_name: str
+    :return: LocationBasedServicesAccount
+    :rtype:
+     ~azure.mgmt.locationbasedservices.models.LocationBasedServicesAccount
+    :raises:
+     :class:`ErrorException<azure.mgmt.locationbasedservices.models.ErrorException>`
+     or :class:`ErrorException<knack.util.CLIError>`
+    """
+    # Retrieve ClientRawResponse from get call
+    out = client.get(resource_group_name, account_name, raw=True)
+    # Handle 404 error
+    if out.response.status_code == 404:
```

Comment:
Many `show` commands use  `error_handler=empty_on_404` kwarg in their command registrations specifically to avoid printing an error message.
This would be a very non-standard way of handling this error. My suggestions:
1) follow suit with other commands and use empty_on_404
2) Do nothing and see if the default error does what you want.
3) Only if the error in 2 is inadequate or results in a stack trace, what a small error handler that accepts the incoming exception, processes it, and raise a CLIError.